### ISSUE TITLE
JsonEventSerializer - Headers.GetLastBytes("eventType") throws KeyNotFoundException

### ISSUE THREAD

**Issue by arslanaybars** [joaofbantunes/DebeziumOutboxSample#1]
Hey @joaofbantunes 

```context.Headers.GetLastBytes("eventType")``` throws KeyNotFound exception. I didn't understand well how eventType could be retrieved from SerializationContext. Can you explain this?

Thanks for the great sample.

**Comment by joaofbantunes** [comment]
Hi @arslanaybars,

Tried to run the sample and effectively did come across some issues, didn't really understand why (probably something changed in the tools since I created it 🤷‍♂️).

I didn't get the `KeyNotFoundException` though, but other issues.

I pushed some changes, that are working on my side now, so hopefully they'll also fix things on your end.

>  I didn't understand well how eventType could be retrieved from SerializationContext. Can you explain this?

Not sure I understand your question. `SerializationContext` gives us access to the message headers, which is where Debezium puts a `eventType` entry, as defined in the `debezium-outbox-config.json` file:
```
{
  // ...
  "transforms.outbox.table.fields.additional.placement": "Type:header:eventType",
  // ...
}
```
